charts/zookeeper-operator/templates/_helpers.tpl

```diff
@@ -3564,14 +3559,13 @@ openAPIV3Schema:
               type: object
             ports:
               items:
-                description: ContainerPort represents a network port in a single container.
+                description: ContainerPort represents a network port in a single
+                  container.
                 properties:
                   containerPort:
                     description: Number of port to expose on the pod's IP address.
                       This must be a valid port number, 0 < x < 65536.
                     format: int32
-                    minimum: 1
-                    maximum: 65535
```

Comment:
some of  the openAPI Schema validations got removed. Could you please fix those?